src/Extension/FrontMatter/FrontMatterExtension.php

```diff
@@ -29,7 +30,7 @@ final class FrontMatterExtension implements ExtensionInterface
 
     public function __construct(?FrontMatterDataParserInterface $dataParser = null)
     {
-        $this->frontMatterParser = new FrontMatterParser($dataParser ?? new SymfonyYamlFrontMatterParser());
+        $this->frontMatterParser = new FrontMatterParser($dataParser ?? LibYamlFrontMatterParser::capable() ?? new SymfonyYamlFrontMatterParser());
```

Comment:
I'm not sure how I feel about this particular change; any users who have `ext-yaml` installed would suddenly start using that parser, even if Symfony's is available and was used by default before.  I suppose that's probably fine though, given that both parsers _should_ largely support the same YAML contents and the extension is faster. So maybe this is okay?  WDYT?

Reply:
For me, it will be faster to parse YAML content of many files with an extension than a library.

But if necessary, one can imagine adding a similar method `SymfonyYamlFrontMatterParser::capable(): SymfonyYamlFrontMatterParser` to provide this parser first.

Reply:
:thinking: I think I'm okay with trying your approach since most users probably won't be affected, and if they are, the only change will likely be faster performance.